lib/generators/feature_flag/USAGE

```diff
@@ -0,0 +1,8 @@
+Description:
```

Comment:
I didn't bother changing this yet. If we go ahead with this proposed solution I'll obviously do that.